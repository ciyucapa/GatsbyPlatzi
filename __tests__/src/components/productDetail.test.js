import React from "react"
import renderer from "react-test-renderer"
import ProductDetail from '../../../src/components/productDetail'

describe("Starts", () => {
    it("renders correctly", () => {
      const tree = renderer
        .create(<ProductDetail 
                    Object={{
                        id: 'sku', 
                        image:'https://test.com/image',
                        price: 1000,
                        product: {
                            name: 'test',
                            metadata: {
                                color: 'azul',
                                description: 'test',
                                img: 'https://test.com/image',
                                wear: true
                            }
                        }
                    }}
                />)
        .toJSON()
      expect(tree).toMatchSnapshot()
    });
});