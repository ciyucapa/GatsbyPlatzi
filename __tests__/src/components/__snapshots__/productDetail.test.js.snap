// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Starts renders correctly 1`] = `
<div
  className="sc-bxivhb kwbWrb"
>
  <img />
  <div>
    <span
      className="sc-gzVnrw fGfoQr"
    >
      Popular
    </span>
    <h2 />
    <b>
      USD 
      $NaN
    </b>
    <div
      className="sc-VigVT bLIVrL"
      selected={5}
    >
      <span
        onClick={[Function]}
        onKeyDown={[Function]}
        role="button"
        tabIndex="0"
      >
        ★
      </span>
      <span
        onClick={[Function]}
        onKeyDown={[Function]}
        role="button"
        tabIndex="0"
      >
        ★
      </span>
      <span
        onClick={[Function]}
        onKeyDown={[Function]}
        role="button"
        tabIndex="0"
      >
        ★
      </span>
      <span
        onClick={[Function]}
        onKeyDown={[Function]}
        role="button"
        tabIndex="0"
      >
        ★
      </span>
      <span
        onClick={[Function]}
        onKeyDown={[Function]}
        role="button"
        tabIndex="0"
      >
        ★
      </span>
    </div>
    <small />
    <p>
      Cantidad: 
    </p>
    <div
      className="sc-iwsKbI dRnGye"
    >
      <button
        onClick={[Function]}
      >
        -
      </button>
      <input
        disabled={true}
        type="text"
        value={1}
      />
      <button
        onClick={[Function]}
      >
        +
      </button>
    </div>
    <button
      className="sc-gqjmRU dvFDel"
      onClick={[Function]}
    >
      Agregar al carrito
    </button>
  </div>
</div>
`;
